### PR TITLE
minor text edits

### PR DESCRIPTION
**Pull request by rosemarymarano**
Some minor text edits/spelling corrections according to the Oracle Style Guide.

### CODE REVIEW

**Review by mgianatagh — APPROVED** [review]

File: examples/multicluster/hello-helidon/README.md [hunk 1]
```diff
@@ -35,11 +34,11 @@ Apply the `VerrazzanoProject` resource on the admin cluster that defines the nam
 
 ## Testing the example application
 
-Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-application) for testing the Hello World Helidon application in a single cluster use case. Use the managed-cluster `kubeconfig` for testing the example application.
+Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-application) for testing the Hello World Helidon application in a single cluster use case. Use the `managed-cluster` `kubeconfig` for testing the example application.
```
Comment (desagar) [inline examples/multicluster/hello-helidon/README.md:37]: This is referring to the managed cluster's kubeconfig. I think using the hyphenated `managed-cluster` here isn't quite right. 
That hyphenated thing is the name of a VZ profile, and that has nothing to do with the kubeconfig. @mgianatagh do you agree?

File: examples/multicluster/hello-helidon/README.md [hunk 2]
```diff
@@ -35,11 +34,11 @@ Apply the `VerrazzanoProject` resource on the admin cluster that defines the nam
...
 
 ## Troubleshooting
 
-Follow the [instructions](../../hello-helidon/README.md/#troubleshooting) for troubleshooting the Hello World Helidon application in a single cluster use case. Use the managed-cluster `kubeconfig` for troubleshooting the example application.
+Follow the [instructions](../../hello-helidon/README.md/#troubleshooting) for troubleshooting the Hello World Helidon application in a single cluster use case. Use the `managed-cluster` `kubeconfig` for troubleshooting the example application.
```
Comment (desagar) [inline examples/multicluster/hello-helidon/README.md:41]: see comment above about a similar usage.

File: examples/multicluster/hello-helidon/README.md (hunk 1 above)
Comment (mgianatagh) [reply examples/multicluster/hello-helidon/README.md:37]: So you are suggesting to replace "`managed-cluster`" and "managed  cluster"?  Similar to line 7 above.  That sounds ok to me.

Comment (desagar) [reply examples/multicluster/hello-helidon/README.md:37]: yes. I am suggesting replacing "`managed-cluster` `kubeconfig`" with:
either "managed cluster `kubeconfig`"
or "managed cluster's `kubeconfig`"

**Review by desagar — APPROVED** [review]

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**